### DISCUSSION TITLE
Overseerr support

### DISCUSSION THREAD

**Discussion by 2fst4u** [Cleanarr/Cleanarr#5] [Ideas]
**Describe the solution you'd like**
Really awesome idea! Any chance of overseerr support?

**Describe alternatives you've considered**
Ombi is ok but a lot of people are jumping ship to overseerr now.

**Additional context**
The ability to remove things after a requester has watched them is exactly what I need!

**Comment by baty0man** [comment]
+1 for overseerr.

**Comment by lbarnett83** [comment]
Another +1 for this

**Comment by appiekap653** [comment]
Yes! Overseerr is definitely on the menu. I will try to write as many plugins possible beforehand and overseerr will be one of them. To make this as open as possible there will be an option for everyone to create their own plugins to support even more different software as media servers, histories providers and request providers.

**Comment by baty0man** [comment]
Hi, is this still in the work? This project looks awesome but yeah without overseer support I can't use it

**Reply by appiekap653** [reply]
Yes it is in the making but due to severe health issues I'm afraid I won't be able to finish it in time. I'm hoping there will be someone else who will finish it.

**Reply by 2fst4u** [reply]
Really sorry to hear that. Maybe we could get the Dev of overseerr to get on board and help with that part.

**Reply by appiekap653** [reply]
As long as there is an API I can hook into it won't be difficult to retrieve the needed information from Overseerr.

**Reply by 2fst4u** [reply]
The documentation is here apparently. I hope it has all the information.

https://api-docs.overseerr.dev/

**Reply by appiekap653** [reply]
Yes that is all that is needed

https://api-docs.overseerr.dev/#/request

You can get all the requested media for all users to compare if the requestee has watched it before you delete the media.

**Comment by nmbrg** [comment]
almost a year since last sign of life in this discussion, so i'll just throw my +1 on this idea in here.

**Comment by aniqueta** [comment]
I just came across this: https://github.com/Supergamer1337/media-cleaner

And before discovering these tools, I posted a feature request here: https://github.com/sct/overseerr/issues/3386
